cpp/oneapi/dal/algo/svm/backend/cpu/train_cls_kernel.cpp

```diff
@@ -131,6 +131,26 @@ static result_t call_multiclass_daal_kernel(const context_cpu& ctx,
         .set_support_indices(table_support_indices);
 }
 
+template <typename Float>
+inline auto convert_to_daal_labels(const table& labels,
```

Comment:
I think it is better not to create one more function but to fix `convert_labels()` function. You can modify it the way it works:
1. Determine the values used to represent binary labels
2. If these values are in the set {(0,1), (1,0), (-1,1), (1,1)}, then use `convert_to_daal_table` function
3. Otherwise, use `convert_to_daal_homogen_table`

This way is more transparent, and do not require always to create an additional array of `arr_new_label`